### PR TITLE
Fixes #522: Refactored RemoveModifierQuickFix and bring the test cases ready

### LINKED ISSUE #522: Transform the RemoveModifierConflictQuickFix from LSP4Jakarta to implement the code action participant extension point.
See the initial work completed here: https://github.com/OpenLiberty/liberty-tools-intellij/issues/306.

In addition to refactoring the code, the code action participants will need to be added to the plugin.xml and the relevant Jakarta EE code action tests should be enabled.

### PR DESCRIPTION
**Pull request by dessina-devasia**
Fixes #522
Refactored RemoveModifierQuickFix and bring the test cases ready
Bring the quick fix working for RemoveStaticModifierQuickFix, RemoveAbstractModifierQuickFix and RemoveFinalModifierQuickFix

### CODE REVIEW

File: src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/JakartaCodeActionHandler.java [hunk 1]
```diff
@@ -36,9 +36,6 @@
 import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.servlet.*;
 import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.websocket.AddPathParamQuickFix;
 import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.websocket.WebSocketConstants;
-import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.codeAction.proposal.quickfix.RemoveAbstractModifierQuickFix;
-import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.codeAction.proposal.quickfix.RemoveFinalModifierQuickFix;
-import io.openliberty.tools.intellij.lsp4jakarta.lsp4ij.codeAction.proposal.quickfix.RemoveStaticModifierQuickFix;
```
Comment (mrglavas) [inline src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/JakartaCodeActionHandler.java:41]: JakartaCodeActionHandler is not being used anymore. It's not necessary to make changes to it.

**Review by mrglavas — APPROVED** [review]
Looks good. Thanks.

File: src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/JakartaCodeActionHandler.java (hunk 1 above)
Comment (aparnamichael) [reply src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/JakartaCodeActionHandler.java:41]: @dessina-devasia Please revert this file changes.

Comment (dessina-devasia) [reply src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/JakartaCodeActionHandler.java]: Reverted changes in JakartaCodeActionHandler

**Review by aparnamichael — APPROVED** [review]
Looks good to me

**Pull request merged**